bundle/Command/IndexPageContentCommand.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaSearchExtraBundle\Command;
+
+use Ibexa\Contracts\Core\Persistence\Handler as PersistenceHandler;
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\Content\Query;
+use Ibexa\Contracts\Core\Repository\Values\Filter\Filter;
+use Ibexa\Contracts\Core\Search\Handler as SearchHandler;
+use Netgen\IbexaSearchExtra\Exception\IndexPageUnavailableException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Ibexa\Contracts\Core\Repository\Values\Content\ContentList;
+
+use function count;
+use function explode;
+
+class IndexPageContentCommand extends Command
```

Comment:
Use `SymfonyStyle` here, for example see https://github.com/netgen/site-bundle/blob/master/bundle/Command/MoveContentTypeCommand.php#L27